src/content/fr/fundamentals/performance/rendering/index.md

```diff
@@ -112,8 +112,7 @@ et le rendu final va devoir être réagencé avec ces autres élements.
 
 ### 2. JS / CSS > Style > Rendu> Composition
 
-<img src="images/intro/frame-no-layout.jpg" alt="The  pixel pipeline without
-layout.">
+<img src="images/intro/frame-no-layout.jpg" alt="Le pipeline pixels sans mise en page.">
```

Comment:
pipeline de pixels?

Reply:
I changed for "pipeline de pixels" in the entire translation.